src/initializers/kafka/kafka.ts

```diff
@@ -179,7 +179,6 @@ function getAuthOptions(options: {
 }) {
   const { key, cert, ca } = options.certificates || {};
   if (key && cert && ca) return { ssl: { ...options.certificates, ca: flatten([ca]) } };
-
   const { username, password } = options.sasl || {};
-  if (username && password) return { sasl: options.sasl, ssl: options.ssl };
+  if (username && password) return { sasl: {...options.sasl }, ssl: options.ssl };
```

Comment:
❓ Was this needed?

Reply:
No it was a leftover from a reverted refactoring...Fixed in https://github.com/Workable/orka/pull/384/commits/e1bc8c05291ae760a6622fc09e8f96df986b5131